### PR TITLE
docs: edit oa-cli readme

### PR DESCRIPTION
**Pull request by rollerchloe**
## Summary

To update a remote file in the documentation website which retrieve information from this readme, changes are proposed.

## Changes

- Mainly editing the content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -510,23 +709,43 @@ open-attestation dns txt-record get --location example.openattestation.com --net
...
 ℹ  info      Wallet with public address 0x6348a96D96D56665C2E9619f81355918779f3d9E successfully created. Find more details:
 ✔  success   Wallet successfully saved into /path/to/tmp
 ```
 
-Encrypting a wallet (see [below](#providing-the-wallet) to find out how to provide the key)
+#### Encrypting a wallet 
+
+Using the `wallet encrypt` command, you can encrypt a wallet.
+
+>**Note:** If you want to provide the wallet private key, see the ["Providing the wallet"](#providing-the-wallet) section.
```
Comment (yapyuyou) [inline README.md:738]: I noticed we link to this article in almost every command. 

1. Do you think there is a better way to do this? 
2. If not, the current line is `See more options [here](#providing-the-wallet).` which I think sounds a bit vague so can we rephrase it to something clearer? e.g. `For more ways to provide the wallet, see [here](#providing-the-wallet).`

File: README.md [hunk 2]
```diff
@@ -106,397 +113,589 @@ Please read the information above to understand the table
...
+```
 ✔  success   Token registry deployed at 0x4B127b8d5e53872d403ce43414afeb1db67B1842
 ```
 
-#### Issue document to token registry
+#### Issuing a document to token registry
+
+The `token-registry issue` command issues a hash to a token registry deployed on the blockchain. The `--tokenId` option indicates the document hash, and the `--to` option indicates the title escrow address to which the document is mapped.
+
+<!--Flag: The syntax below didn't mention the `--to` option. The command example mentions `--to` option, but did not show how to use `--beneficiary` or `--holder` options.-->
```
Comment (yapyuyou) [inline README.md:435]: Thanks for pointing this out! Let me check on it and get back to you

**Review by yapyuyou — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,66 +1,69 @@
-# Open Attestation (CLI)
+# OpenAttestation (CLI)
 
-This CLI tool in the [Open Attestation CLI](https://github.com/Open-Attestation/open-attestation-cli) repository turns .json documents into any open-attestation verifiable documents. It applies the OpenAttestation algorithm to produce a hash of the json document and then creates a file with the data and proof of integrity.
+This command line interface tool in the [OpenAttestation CLI](https://github.com/Open-Attestation/open-attestation-cli) repository turns `.json` documents into OpenAttestation verifiable documents. It applies OpenAttestation algorithm to produce a hash of the `.json` document, and then creates a file with the data and proof of integrity.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:3]: I thought not all "the" need to be spelt out. but sure, can add it back.

[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:3]: recovered "the OpenAttestation algorithm".

File: README.md [hunk 4]
```diff
@@ -1,66 +1,69 @@
...
 
 ## Installation
 
+There are three ways to install the CLI, including binary, NPM, and NPX.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:7]: updated to: "There are three ways to access the CLI, including binary installation, NPM installation, and NPX execution."

File: README.md [hunk 5]
```diff
@@ -1,66 +1,69 @@
...
 
 ```bash
 npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
 ```
 
-> In all the guides, we will refer to the CLI as `open-attestation` when running a command. That means we will assume the CLI is available in your execution path. If it's not the case, you will to change `open-attestation` by the full path to the executable.
+No matter which way you choose, once the installation process is complete, a configuration folder will be created in the `~/.config/open-attestation/` folder.
 
-> A configuration folder will be created in the `~/.config/open-attestation/`
+>**Important:** On this website, the CLI is referred to as `open-attestation` when you run a command. The assumption is the CLI is available in your execution path. If not, you must change `open-attestation` to reflect the full path to the executable.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:34]: updated

File: README.md [hunk 6]
```diff
@@ -1,66 +1,69 @@
...
-You can also opt to use npx:
+### NPX 
+You can also opt to use `npx` for the installation:
 
 ```bash
 npx -p @govtechsg/open-attestation-cli open-attestation <arguments>
 ```
 
-> In all the guides, we will refer to the CLI as `open-attestation` when running a command. That means we will assume the CLI is available in your execution path. If it's not the case, you will to change `open-attestation` by the full path to the executable.
+No matter which way you choose, once the installation process is complete, a configuration folder will be created in the `~/.config/open-attestation/` folder.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:32]: updated

File: README.md [hunk 7]
```diff
@@ -1,66 +1,69 @@
...
 ```bash
 npm install -g @govtechsg/open-attestation-cli
 ```
 
-The above command will install the open-attestation CLI to your machine. You will need to have node.js installed to be able to run the command.
+The above command will install the open-attestation CLI to your machine. Be sure to have `node.js` installed, so that you can run the command.
 
-You can also opt to use npx:
+### NPX 
+You can also opt to use `npx` for the installation:
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:26]: updated

File: README.md [hunk 8]
```diff
@@ -106,397 +113,589 @@ Please read the information above to understand the table
...
+The response looks like: 
+```
 ✔  success  Batch Document Root: 0xf51030c5751a646284c898cff0f9d833c64a50d6f307b61f2c96c3c838b13bfc
 ```
 
-The JSON schema parameter also allow for http endpoint returning valid JSON schema:
+##### HTTP endpoint in the `--schema` option 
 
-Example:
+With the `schema` parameter, an HTTP endpoint will return a valid JSON schema.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:224]: updated

File: README.md [hunk 9]
```diff
@@ -106,397 +113,589 @@ Please read the information above to understand the table
...
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --open-attestation-v3
+```
+Alternatively, you can use the short form `--oav3` to replace `--open-attestation-v3` in the command:
+
+```bash
 open-attestation wrap ./examples/raw-documents/ ./examples/wrapped-documents/ --oav3
 ```
 
-> **_NOTE:_** For transferable records, you should wrap them individually as each of them would be minted to a unique title escrow that represents the beneficiary and holder entities of the document. For more information about title escrow, refer [here](https://www.openattestation.com/docs/integrator-section/transferable-record/title-escrow).
+>**Note:** You should wrap Transferable Records individually, as each of them will be minted to a unique title escrow that represents the document's beneficiary and holder entities. For more information about title escrow, see [here](https://www.openattestation.com/docs/integrator-section/transferable-record/overview).
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:293]: updated

File: README.md [hunk 10]
```diff
@@ -106,397 +113,589 @@ Please read the information above to understand the table
...
 
 ### Unwrapping documents
 
-This command processes a document in the input directory. It will unwrap the wrapped document to its raw document form to be displayed on the console.
+The `unwrap` command processes a document in the input directory. It will unwrap the document that has been wrapped to its raw document form, and display it in the console.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:297]: updated

File: README.md [hunk 11]
```diff
@@ -551,20 +781,28 @@ open-attestation wallet decrypt wallet.json
 - private key ....
 ```
 
-### Providing the wallet
+#### Providing the wallet
+
+When interacting with blockchain, you may need to provide a way to access your wallet. When the wallet is required, all functions will provide multiples ways for you to provide the wallet information:
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:786]: updated

File: README.md [hunk 12]
```diff
@@ -582,43 +820,52 @@ rm ./examples/sample-key
...
-Command options to take note:
+You can use the following options in this command:
 
-- `--output-dir` option specify which folder the config file will be created in.
+- `--output-dir` option specifies in which folder the configuration file will be created.
 - `--encrypted-wallet-path` option indicates a path to an [encrypted wallet](https://docs.ethers.io/v5/api/signer/#Wallet-encrypt).
 
-<!-- - `--config-template-url` option to provide a path to reference a config template file hosted on a public url.
-- `--config-template-path` option to provide a path to reference a config template file locally. -->
+- `--config-template-url` option provides a path to reference a configuration template file hosted on a public URL. To learn more, see [Method 1](#method-1-using-the-config-template-url-option-recommended).
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:834]: deleted the links

File: README.md [hunk 13]
```diff
@@ -582,43 +820,52 @@ rm ./examples/sample-key
...
+
+- `--config-template-path` option provides a path to reference a config template file locally. To learn more, see [Method 2](#method-2-using-the-config-template-path-option).
 
-There are 2 ways of using this command to generate a config file, both in which, will return a new config file with sandbox DNS, updated document store and updated token registry.
+There are two ways of using this command to generate a configuration file. Both ways will return a new configuration file with the sandbox DNS, updated document store, and updated token registry.
```
[the remark this answers is not in the document]
Comment (rollerchloe) [reply README.md:838]: rewritten

File: README.md (hunk 1 above)
Comment (rollerchloe) [reply README.md:738]: updated all

File: README.md (hunk 2 above)
Comment (rollerchloe) [reply README.md:435]: updated the command & response

File: README.md [hunk 14]
```diff
@@ -22,16 +22,17 @@ npm install -g @govtechsg/open-attestation-cli
 
 The above command will install the open-attestation CLI to your machine. Be sure to have `node.js` installed, so that you can run the command.
 
-### NPX 
-You can also opt to use `npx` for the installation:
+### NPX
+
+If you only want to execute a few commands, you can also opt to use `npx` for the installation:
```
Comment (yapyuyou) [inline README.md:27]: npx cannot be used for installation

```suggestion
If you only want to execute a few commands, you can also opt to use `npx`:
```

File: README.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (yapyuyou) [reply README.md]: added a new comment for this

File: README.md [hunk 15]
```diff
@@ -441,17 +444,20 @@ open-attestation token-registry issue --network <NETWORK> --address <TOKEN_REGIS
...
 
 ```bash
-open-attestation token-registry mint --network sepolia --address 0x6133f580aE903b8e79845340375cCfd78a45FF35 --tokenId 0x10ee711d151bc2139473a57531f91d961b639affb876b350c31d031059cdcc2c --to 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --tokenId 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea --beneficiary 0x6FFeD6E6591b808130a9b248fEA32101b5220eca --holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca -n sepolia -f key.txt
 ```
 The response looks like:
 
 ```
-✔  success   Token with hash 0x10ee711d151bc2139473a57531f91d961b639affb876b350c31d031059cdcc2c has been issued on 0x6133f580aE903b8e79845340375cCfd78a45FF35 with the initial recipient being 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+✔  success   Token with hash 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea has been issued on 0x1E63411DC2fCd6Fab5EE938622f5f6A390F48272 with the initial recipient being 0x6FFeD6E6591b808130a9b248fEA32101b5220eca and initial holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca
```
Comment (yapyuyou) [inline README.md:460]: I think there is an error here as well as in the current `Issuing Transferable Record` page. The address supplied above should be the same as the one in the expected output.

e.g. in the above command we pass in 
```
... open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 ...
```

the response should be
```
... has been issued on 0x8431012Bc040942B59e3C5bf428221eab0b2f723 ...
```
instead of `0x1E63411 ...`

File: README.md [hunk 16]
```diff
@@ -441,17 +444,20 @@ open-attestation token-registry issue --network <NETWORK> --address <TOKEN_REGIS
...
 
-The following command is a recommended example with the private key set in the `OA_PRIVATE_KEY` environment variable. 
+In the example below: 
+* You will use 0x6FFeD6E6591b808130a9b248fEA32101b5220eca as beneficiary and holder.
+* You will need to replace the value above with a wallet address you control, to be able to perform different actions on the transferable records later.
+* The `key.txt` file stores the private key to token registry.
 
 ```bash
-open-attestation token-registry mint --network sepolia --address 0x6133f580aE903b8e79845340375cCfd78a45FF35 --tokenId 0x10ee711d151bc2139473a57531f91d961b639affb876b350c31d031059cdcc2c --to 0xB26B4941941C51a4885E5B7D3A1B861E54405f90
+open-attestation token-registry issue -a 0x8431012Bc040942B59e3C5bf428221eab0b2f723 --tokenId 0x0d9839a8034cb783d98bd57bcbaafb4dc3614c4193d2edf8a655c1ec6635b7ea --beneficiary 0x6FFeD6E6591b808130a9b248fEA32101b5220eca --holder 0x6FFeD6E6591b808130a9b248fEA32101b5220eca -n sepolia -f key.txt
```
Comment (yapyuyou) [inline README.md:455]: For clarity lets use the verbose option `--address` instead of `-a`

File: README.md (hunk 2 above)
Comment (yapyuyou) [reply README.md]: thanks Chloe! Added some comments on this

File: README.md (hunk 14 above)
Comment (rollerchloe) [reply README.md:27]: removed

File: README.md (hunk 15 above)
Comment (rollerchloe) [reply README.md:460]: updated in both places

File: README.md (hunk 16 above)
Comment (rollerchloe) [reply README.md:455]: updated in this readme & documentation-website PR 193

**Review by yapyuyou — APPROVED** [review]
